guildwars2/characters.py

```diff
@@ -459,3 +727,43 @@ def get_crafting(self, character):
             discipline = crafting["discipline"]
             craft_list.append("Level {} {}".format(rating, discipline))
         return craft_list
+
+    def search_lvl_to_increase(self, level: int, lvl_dict):
+        for increase, lvl in lvl_dict.items():
+            if lvl[0] <= level <= lvl[1]:
+                if level < 11:
+                    return increase
+                elif level % 2 == 0:
+                    return increase
+                else:
+                    return 0
+
+    def calc_base_lvl(self, level: int, acc_baselvl: int, lvl_dict):
+        # Recursive call of search_lvl_to_increase
+        # Calculating the base primary attributes depending on char lvl
+        if level == 1:
+            acc_baselvl += 37
+            return acc_baselvl
+        else:
+            new_acc = acc_baselvl + self.search_lvl_to_increase(
+                level, lvl_dict)
+            new_lvl = level - 1
+            return self.calc_base_lvl(new_lvl, new_acc, lvl_dict)
+
+    def search_lvl_to_health(self, level: int, health_dict):
+        for increase, lvl in health_dict.items():
+            if lvl[0] <= level <= lvl[1]:
+                return increase
+
+    def calc_base_health(self, level: int, acc_baselvl: int, health_dict):
```

Comment:
All those helper methods could be internal functions to main command function

Reply:
Moved helper functions inside function